loopchain/blockchain/blocks/v0_2/block_builder.py

```diff
@@ -23,3 +24,18 @@ def build_block_header_data(self):
         header_data = super().build_block_header_data()
         header_data["is_complain"] = self.is_complain
         return header_data
+
+    def from_(self, block: 'Block'):
+        super().from_(block)
+
+        header: BlockHeader = block.header
+        self.next_leader = header.next_leader
+        self.commit_state = header.commit_state
+        self.merkle_tree_root_hash = header.merkle_tree_root_hash
+        self.fixed_timestamp = header.timestamp
+
+        self._timestamp = header.timestamp
+        self.is_complain = header.is_complain
```

Comment:
How about changing `is_complain` to `complained`?
Tow verbs(is(verb) complain(verb)) are weird.